vagrant/provision.sh

```diff
@@ -24,3 +24,44 @@ curl -LJ "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" -o go.linux-
 
 # Install ansible
 pip3 install ansible
+
+# Add puppet hosts
+tee -a /etc/hosts << END
+192.168.79.13 puppetserver.example.com puppet
+192.168.79.13 agent.example.com
+END
+
+# Install puppet server & puppet agent
+wget https://apt.puppetlabs.com/puppet6-release-focal.deb
+dpkg -i puppet6-release-focal.deb
+apt-get update -y
+apt-get install puppetserver -y
+apt-get install puppet-agent -y
```

Comment:
```suggestion
apt-get install puppetserver puppet-agent -y
```

Why not together? Should be a touch faster